packages/manager/modules/support/src/modal/modal.html

```diff
@@ -0,0 +1,29 @@
+<oui-modal
+    class="beta-modal"
+    data-on-dismiss="$ctrl.goBack()"
+    data-secondary-action="$ctrl.goBack()"
+>
+    <div class="row d-flex justify-content-center align-items-center">
+        <div class="col-6">
+            <h1 data-translate="modal_beta_help_center_description"></h1>
+            <p data-translate="modal_beta_help_center_please_note"></p>
+            <oui-button ng-click="$ctrl.onClick()" variant="primary">
+                <span data-translate="modal_beta_help_center_access"></span
+            ></oui-button>
+
+            <oui-button
+                on-click="$ctrl.goBack()"
+                variant="link"
+                variant-nav="next"
+            >
+                <span data-translate="modal_beta_help_center_later"></span>
+            </oui-button>
+        </div>
+        <div class="col-6">
+            <img
+                data-ng-src="{{ $ctrl.illustration }}"
+                alt="Beta image modal"
```

Comment:
I think it's a good practice to have put this.
So, we can add a French alternative description.

`alt="{{ 'ma_description' | translate }}"`